proto/.clang-format

```diff
@@ -0,0 +1,8 @@
+---
+BasedOnStyle: Google
+IndentWidth: 4
+---
+Language: Proto
+ColumnLimit: 100
```

Comment:
I'm anti line-length so I vote to remove this entirely.

Reply:
I really want to bump our go line length cap to 120.

Reply:
hmm I'm for having a length cap. I was between picking 80 or 100 if you prefer a higher limit stick with 100 to be consistent with our Go? :thinking: